test/conformance/errorcondition_test.go

```diff
@@ -117,80 +117,97 @@ func TestContainerErrorMsg(t *testing.T) {
 // https://github.com/knative/serving/blob/master/docs/spec/errors.md
 // for the container crashing scenario.
 func TestContainerExitingMsg(t *testing.T) {
-	t.Parallel()
-	clients := setup(t)
-
-	names := test.ResourceNames{
-		Config: test.ObjectNameForTest(t),
-		Image:  "failing",
-	}
-
 	const (
 		// The given image will always exit with an exit code of 5
 		exitCodeReason = "ExitCode5"
 		// ... and will print "Crashed..." before it exits
 		errorLog = "Crashed..."
 	)
 
-	t.Logf("Creating a new Configuration %s", names.Image)
-
-	// This probe is crucial for having a race free conformance test. It will prevent the
-	// pod from becoming ready intermittently.
-	probe := &corev1.Probe{
-		Handler: corev1.Handler{
-			HTTPGet: &corev1.HTTPGetAction{},
+	tests := []struct {
+		Name           string
+		ReadinessProbe *corev1.Probe
+	}{{
+		Name: "http",
+		ReadinessProbe: &corev1.Probe{
+			Handler: corev1.Handler{
+				HTTPGet: &corev1.HTTPGetAction{},
```

Comment:
Is a "Path" not required here?

Reply:
The path is optional. Since the image is failing, I do not think it's providing any extra context by adding the path since nothing is exposed in the failing image.  https://github.com/knative/serving/blob/master/vendor/k8s.io/api/core/v1/types.go#L1878-L1880